.travis.yml

```diff
@@ -19,6 +19,7 @@ install:
   - bundle check --path vendor/.bundle/ || bundle install --path vendor/.bundle/
   - yarn check || yarn install
   - bundle exec rake db:setup || bundle exec rake db:migrate
+  - rm -rf test/reports
```

Comment:
There's no need to add this additional step to Travis, causing the build to be slower, I think you could just add test/reports to .gitignore (if its not there already), then remove this folder if its commited. But let's hear from @jywarren.

Reply:
It's already in .gitignore 
@alaxalves Can you help removing it from git tracking?